### PR TITLE
[1LP][RFR] Fixed Embedded Ansible CRUD test file by re-ordering sub-tests

### PR DESCRIPTION
**Pull request by sbulage**
PRT Run:

{{ pytest: cfme/tests/ansible/test_embedded_ansible_crud.py --long-running }}

Fixes #8702

### CODE REVIEW

**Comment by izapolsk** [comment]
@sbulage, please rebase your PR.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by sbulage** [comment]
@izapolsk I will cross-check locally why it is failing for disable (will check service status.)

File: cfme/tests/ansible/test_embedded_ansible_crud.py [hunk 1]
```diff
@@ -34,44 +34,19 @@ def test_embedded_ansible_enable(embedded_appliance):
     assert wait_for(lambda: embedded_appliance.is_embedded_ansible_running, num_sec=30)
     assert wait_for(lambda: embedded_appliance.rabbitmq_server.running, num_sec=30)
     assert wait_for(lambda: embedded_appliance.nginx.running, num_sec=30)
+    assert wait_for(
+        lambda: embedded_appliance.supervisord.status == "active", num_sec=180
+    )
+    assert wait_for(
+        lambda: embedded_appliance.rabbitmq_server.status == "active", num_sec=80
```
Comment (digitronik) [inline cfme/tests/ansible/test_embedded_ansible_crud.py]: Q: after `status` check do you think above `rabbitmq_server.running` check required?

File: cfme/tests/ansible/test_embedded_ansible_crud.py [hunk 2]
```diff
@@ -133,3 +106,38 @@ def test_embedded_ansible_logs(embedded_appliance):
...
+            lambda: embedded_appliance.rabbitmq_server.status == "inactive", num_sec=80
+        )
+        assert wait_for(
+            lambda: embedded_appliance.nginx.status == "inactive", num_sec=30
+        )
+    else:
+
+        @wait_for_decorator(num_sec=300)
+        def is_ansible_pod_stopped():
+            # todo: implement appropriate methods in appliance
```
Comment (digitronik) [inline cfme/tests/ansible/test_embedded_ansible_crud.py]: Required: Please fallow TODO guideline https://github.com/ManageIQ/integration_tests/blob/master/docs/guides/dev_guide.rst

File: cfme/tests/ansible/test_embedded_ansible_crud.py (hunk 1 above)
Comment (sbulage) [reply cfme/tests/ansible/test_embedded_ansible_crud.py]: Earlier `.running` used to return `True` or `False`, so my intention to check existing status of the service.

File: cfme/tests/ansible/test_embedded_ansible_crud.py (hunk 2 above)
Comment (sbulage) [reply cfme/tests/ansible/test_embedded_ansible_crud.py]: @digitronik thanks for pointing this out, I am gonna placing this method in `IPAppliance`.

**Comment by sbulage** [comment]
@izapolsk Placed `is_ansible_pod_stopped` method into `IPAppliance`.

File: cfme/tests/ansible/test_embedded_ansible_crud.py [hunk 3]
```diff
@@ -133,3 +99,30 @@ def test_embedded_ansible_logs(embedded_appliance):
     # Retrieving setup log file from list and asserting with length
     # Setup log file contains date/time string in it.
     assert "setup" in diff[0]
+
+
+@pytest.mark.tier(3)
+def test_embedded_ansible_disable(embedded_appliance):
```
Comment (digitronik) [inline cfme/tests/ansible/test_embedded_ansible_crud.py]: @sbulage I am suspect its order problem. I think better solution of this to add small fixture like `ensure_enabled` or `ensure_disable` which will reduce your test dependency.

Comment (sbulage) [reply cfme/tests/ansible/test_embedded_ansible_crud.py]: Okay @digitronik I will update it.

Comment (sbulage) [reply cfme/tests/ansible/test_embedded_ansible_crud.py]: Updated.

File: cfme/tests/ansible/test_embedded_ansible_crud.py [hunk 4]
```diff
@@ -3,17 +3,15 @@
 from cfme import test_requirements
 from cfme.utils.log import logger
 from cfme.utils.wait import wait_for
-from cfme.utils.wait import wait_for_decorator
 
 pytestmark = [
     pytest.mark.ignore_stream("upstream"),
-    test_requirements.ansible,
+    test_requirements.ansible
```
Comment (mshriver) [inline cfme/tests/ansible/test_embedded_ansible_crud.py]: Why remove trailing commas here? (and below)

**Pull request merged**